src/app/modules/item/pages/item-content/item-content.component.html

```diff
@@ -0,0 +1,4 @@
+<ng-container *ngIf="item$ | async as item">
+  <alg-chapter-detail *ngIf="item.type === 'Chapter'">
```

Comment:
We discussed yesterday other cases where the this component should not be displayed (or behave correctly in this case at least).